Wabbajack.Common/DynamicIniData.cs

```diff
@@ -35,7 +36,7 @@ public override bool TryGetMember(GetMemberBinder binder, out object result)
             return true;
         }
         
-        public override bool TryGetIndex(GetIndexBinder binder, object[] indexes, out object result)
+        public override bool TryGetIndex(GetIndexBinder binder, object[] indexes, [MaybeNullWhen(false)] out object result)
```

Comment:
This is an example of MaybeNullWhen concepts for use in TryGet concepts.
This allows you to assign null to a normally non-nullable out parameter.  The compiler also notices this on the user-end and will warn the user when they use the variable without checking the return bool first.